.github/README.md

```diff
@@ -39,6 +40,81 @@ cd  # in your home directory
 git clone https://github.com/torvalds/linux.git
 ```
 
+If you are compiling the kernel for the first time, install some of the packages required:
+
+On a Debian/Ubuntu distribution:
+
+
+```
+sudo apt install -y \
+  qtdeclarative5-dev \
```

Comment:
the dependency to qtdeclarative5-dev does not seem right to me, it's probably qtbase5-dev